RecoTracker/TkDetLayers/src/TkDetUtil.cc

```diff
@@ -40,7 +50,8 @@ namespace tkDetUtil {
       auto y1 = hori ? yc - maxDistance.y() :  xc - maxDistance.x();
```

Comment:
it is of course intended. if xl-yl and xt,yt are the bottom-left and top-right corner of the rectangle
 I can make you the drawing if you want...  (the window can never be between [xl-yt:xt-yt].
if crossing quadrant is  [xl-yl:xt-yl] . if fully contained in one quadrant is  [xl-yl:xt-yt] or the opposite..
(and in any case I run for many events and verified identity with the 4corner-computation (remove the return, decomment the printout...)
(in any case this code was introduced in #12749 and fully validated)